### ISSUE TITLE
SketchUp extension unable to open OpenStudio 1.2.0-alpha

### ISSUE THREAD

**Issue by Ski90Moo** [openstudiocoalition/openstudio-sketchup-plugin#74]
In the extension preferences, when assigning the file path to the latest OS version 1.2.0-alpha, it is unable to find a module:

![2021-05-25](https://user-images.githubusercontent.com/69771412/119540050-d0e31900-bd41-11eb-8ea8-070f7553aeab.png)

Now, the extension in SketchUp does not load, and I am unable to go back and assign the file path to the older version of OpenStudio.

**Comment by macumber** [comment]
What version of SketchUp are you trying to load OS version 1.2.0-alpha in?  The 1.2.0 version will only work with SketchUp 2021 and above.

**Comment by macumber** [comment]
If you temporarily rename the OpenStudio installation folder from `openstudioapplication-1.2.0-alpha` to something like `temp-openstudioapplication-1.2.0-alpha` and start SketchUp it should allow you to pick another OpenStudio install that works with your version of SketchUp.

**Comment by Ski90Moo** [comment]
Ok, thank you.  Yes, I was using SketchUp 2020.

**Comment by macumber** [comment]
I've updated the version compatibility matrices to include 1.2.0

**Comment by Ski90Moo** [comment]
Ok, I just installed SketchUp 2021 and tried to install the extension with the folder path to several different folder, but it keeps giving me the same error.  Here are the folders I tried:

openstudioapplication-1.1.0
openstudioapplication-1.1.0-rc3temp
openstudioapplication-1.1.1-alpha

If I rename the folders with temp, then it does prompt me for a different path.  The problem is that none of the paths work.
